src/app/public-view/components/content/components/content-text-section/content-text-section.component.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/app/public-view/components/content/project-content.component.scss';
+
+.line-under-text {
+  height:3px;
```

Comment:
mozno by som dal na %

Reply:
Urcite na % ked to otvoris na 4k tak budes vidiet taku bodecku len :D